README.rst

```diff
@@ -69,3 +69,43 @@ We use `Travis`_ coupled with `Coveralls`_ as continious integration tools.
 
 .. image:: https://ga-beacon.appspot.com/UA-87461-7/django-dbbackup/home
         :target: https://github.com/igrigorik/ga-beacon
+
+Tests
+=====
+
+Tests are stored in :mod:`dbbackup.tests` and for run them you must launch:
+
+::
+
+    python runtests.py
```

Comment:
That sounds cool.. didn't know that :) Recommend to amend the Makefile `test` target to reflect this also, such that `make test` works, too.